packages/next/client/head-manager.js

```diff
@@ -47,6 +47,15 @@ export default class HeadManager {
   updateElements (type, components) {
     const headEl = document.getElementsByTagName('head')[0]
     const headCountEl = headEl.querySelector('meta[name=next-head-count]')
+    if (!headCountEl) {
+      if (process.env.NODE_ENV !== 'production') {
+        console.warn(
+          'Warning: next-head-count is missing. https://err.sh/next.js/next-head-count-missing'
+        )
+      }
+      return
```

Comment:
Seems like this is an edge case right 🤔 If it does not exist something is wrong with the app 🤔

Reply:
Definitely an edge case. I figured a warning might be useful since the behavior was changed, even if the requirements technically didn’t.

Reply:
I think it'd be good to throw or not have the check